File: 02_devApplication_IOS_avec_Objective_C/10_UITableVieux_CellulePerso_MasterDetail/10_UITableVieux_CellulePerso_MasterDetail/Gateau.m
//
//  Gateau.m
//  10_UITableVieux_CellulePerso_MasterDetail
//


#import "Gateau.h"

@implementation Gateau

-(id)initNom:(NSString *)nom nomImage:(NSString *)nomImage description:(NSString *)description;
{
    if( self = [super init])
    {
        self.nomImage = nomImage;
        self.nom = nom;
        self.texte = description;
    }
    return self;
}
@end

File: 04_devApplication_IOS_avec_Swift/keke/06_UiTableView_CellulePersonnalisee_MasterDetail/08_UiTableView_CellulePersonnalisee_MasterDetail/Gateau.swift
//
//  Gateau.swift
//  08_UiTableView_CellulePersonnalisee_MasterDetail
//
//  Created by etudiant on 5/11/17.
//  Copyright © 2017 etudiant. All rights reserved.
//

import Foundation

class Gateau
{
    var nomImage: String
    var nom: String
    var description: String
    
    init(nom:String, nomImage:String, description:String)
    {
        self.nom = nom
        self.nomImage = nomImage
        self.description = description
    }
}
